src/components/Textarea/Textarea.js

```diff
@@ -15,6 +15,8 @@ import { useStyles } from '@helpers/hooks/useStyles'
  * @param {string} variant - variant
  * @param {string} size - size
  * @param {object} i18n - props with translated string
+ * @param {bool} isResize - is rezise
+ * @param {bool} maxHeight - max height
```

Comment:
To raczej nie jest bool
No i pojawiły się już tu 3 zmienne number, bool i null dla tego propsa:D